android/app/src/main/java/com/chorestory/module/NetModule.java

```diff
@@ -0,0 +1,34 @@
+package com.chorestory.module;
+
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import javax.inject.Singleton;
+
+import dagger.Module;
+import dagger.Provides;
+import okhttp3.OkHttpClient;
+import retrofit2.Retrofit;
+
+@Module
```

Comment:
I'm gonna look up some tutorials, but I would prefer if we could setup a meeting where you explained Dagger a bit more interactively.

Reply:
big agree